src/main/java/org/jabref/logic/journals/JournalInformationFetcher.java

```diff
@@ -0,0 +1,229 @@
+    private static final Pattern QUOTES_BRACKET_PATTERN = Pattern.compile("[\"\\[\\]]");
+
+    @Override
+    public String getName() {
+        return JournalInformationFetcher.NAME;
+    }
+
+    public Optional<JournalInformation> getJournalInformation(String issnString, String journalName) throws FetcherException {
+        ISSN issn = new ISSN(issnString);
+        String cleanedISSN = "";
+
+        if (issn.isValidFormat() || issn.isCanBeCleaned()) {
+            cleanedISSN = issn.getCleanedISSN();
+        } else {
+            LOGGER.warn(Localization.lang("Incorrect ISSN format"));
+        }
+
+        Optional<JournalInformation> journalInformationOptional = Optional.empty();
+
+        JSONObject postData = buildPostData(cleanedISSN, journalName);
+
+        HttpResponse<JsonNode> httpResponse = Unirest.post(API_URL)
+                                                     .header("Content-Type", "application/json")
+                                                     .body(postData)
+                                                     .asJson();
+
+        if (httpResponse.getBody() != null) {
+            JSONObject responseJsonObject = httpResponse.getBody().getObject();
+            journalInformationOptional = Optional.of(parseResponse(responseJsonObject));
+        }
+
+        return journalInformationOptional;
+    }
+
+    private JournalInformation parseResponse(JSONObject responseJsonObject) throws FetcherException {
+        String title = "";
+        String publisher = "";
+        String coverageStartYear = "";
+        String coverageEndYear = "";
+        String subjectArea = "";
+        String country = "";
+        String categories = "";
+        String scimagoId = "";
+        String hIndex = "";
+        String issn = "";
+        List<Pair<Integer, Double>> sjrArray = new ArrayList<>();
+        List<Pair<Integer, Double>> snipArray = new ArrayList<>();
+        List<Pair<Integer, Double>> docsThisYear = new ArrayList<>();
+        List<Pair<Integer, Double>> docsPrevious3Years = new ArrayList<>();
+        List<Pair<Integer, Double>> citableDocsPrevious3Years = new ArrayList<>();
+        List<Pair<Integer, Double>> citesOutgoing = new ArrayList<>();
+        List<Pair<Integer, Double>> citesOutgoingPerDoc = new ArrayList<>();
+        List<Pair<Integer, Double>> citesIncomingByRecentlyPublished = new ArrayList<>();
+        List<Pair<Integer, Double>> citesIncomingPerDocByRecentlyPublished = new ArrayList<>();
+
+        try {
+            if (responseJsonObject.has("errors")) {
+                JSONArray errors = responseJsonObject.optJSONArray("errors");
+                if (errors != null && errors.length() > 0) {
+                    JSONObject error = errors.getJSONObject(0);
+                    String errorMessage = error.optString("message", "");
+                    LOGGER.error("Error accessing catalog: " + errorMessage);
```

Comment:
Please follow the log pattern here:

```suggestion
                    LOGGER.error("Error accessing catalog: {}", errorMessage);
```

Reply:
Hmm thats strange, the popup is coded to open at the point of the button. I'll check this once the api PR is merged